README.md

```diff
@@ -52,70 +78,37 @@ To see the list commands possible with the CLI, simply ask for help `$# pact --h
-	pactFileWriteMode: <String>          // Control how the pact file is created. One of 'overwrite', 'update' or 'merge'. Defaults to 'overwrite'.
-});
-```
+Mock servers are used by Pact to record interactions and create pact contracts.
 
-### Publish Pacts to a Broker
+#### Create Mock Server
 
 ```js
 var pact = require('@pact-foundation/pact-node');
-var opts = {
-	pactFilesOrDirs: <Array>,        // Array of local Pact files or directories containing them. Required.
-	pactBroker: <String>,            // URL to fetch the provider states for the given provider API. Optional.
-	pactBrokerUsername: <String>,    // Username for Pact Broker basic authentication. Optional
-	pactBrokerPassword: <String>,    // Password for Pact Broker basic authentication. Optional,
-	tags: <Array>,                   // An array of Strings to tag the Pacts being published. Optional
-	consumerVersion: <String>        // A string containing a semver-style version e.g. 1.0.0. Required.
-};
-
-pact.publishPacts(opts).then(function () {
-	// do something
+var server = pact.createServer({
+	...
 });
 ```
 
-### List Mock Servers
+**Options:**
+
+|Parameter | Required?  | Type        | Description |
+|----------|------------|-------------|-------------|
+| `port`   | false |  number          | Port number that the server runs on, defaults to random available port |
+| `host`   | false |  string          | Host on which to bind the server on, defaults to 'localhost'. Supports '0.0.0.0' to bind on all IPv4 addresses on the local machine. |
+| `log`    | false |  string          | File to log output on relative to current working directory, defaults to none |
+| `ssl`    | false |  boolean         | Create a self-signed SSL cert to run the server over HTTPS , defaults to `false` |
+| `sslcert`| false |  string          | Path to a custom self-signed SSL cert file, 'ssl' option must be set to true to use this option, defaults to none |
+| `sslkey` | false |  string          | Path a custom key and self-signed SSL cert key file, 'ssl' option must be set to true to use this, defaults to none |
+| `cors`   | false |  boolean         | Allow CORS OPTION requests to be accepted, defaults to 'false' |
+| `dir`    | false |  string          | Directory to write the pact contracts relative to the current working directory, defaults to none  |
+| `spec`   | false | number           | The pact specification version to use when writing pact contracts, defaults to '1' |
+| `consumer`          | false  | string  | The name of the consumer to be written to the pact contracts, defaults to none |
+| `provider`          | false  | string  |  The name of the provider to be written to the pact contracts, defaults to none |
+| `pactFileWriteMode` | false  | string  | Control how the pact file is created. One of 'overwrite', 'update' or 'merge', defaults to 'overwrite' |
```

Comment:
Instead of `string` should we use TS types? like `"overwrite" | "update" | "merge"` instead?

Reply:
Sure!